Command/LoadDataFixturesDoctrineODMCommand.php

```diff
@@ -93,11 +84,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $dm = $this->getManagerRegistry()->getManager($input->getOption('dm'));
         $ui = new SymfonyStyle($input, $output);
 
-        $dirOrFile = $input->getOption('fixtures');
-        $bundles   = $input->getOption('bundles');
-        $services  = (bool) $input->getOption('services');
-        if (($services && $bundles) || ($services && $dirOrFile) || ($bundles && $dirOrFile)) {
-            throw new InvalidArgumentException('Can only use one of "--bundles", "--fixtures", "--services".');
+        if ((bool) $input->getOption('services')) {
+            @trigger_error(sprintf('The "services" option to the "%s" command is deprecated and will be dropped in DoctrineMongoDBBundle 5.0.', $this->getName()), E_USER_DEPRECATED);
```

Comment:
I've never used fixtures with ODM (and must admit I haven't watched latest fixtures PRs closely) - why can't we remove this already?

Reply:
With 3.6, users will have to use the services option to not have a deprecation message. I want them to only have to worry about removing that once they run 4.0, but still have it work with it instead of adding another BC break.